src/portal/src/app/project/webhook/webhook.ts

```diff
@@ -0,0 +1,63 @@
+import { WebhookEventTypes } from '../../shared/shared.const';
+// {
```

Comment:
delete it

Reply:
`WebhookEventTypes ` is in used.
![image](https://user-images.githubusercontent.com/11702505/62758667-a020e980-bab1-11e9-8bc7-9f3188caa1ed.png)

Reply:
done